WalletWasabi.Backend/wwwroot/faq.html

```diff
@@ -0,0 +1,400 @@
+                                <br>- WHAT WALLETS CAN I USE INSTEAD?
+                            </label>
+                            <div class="card-content">
+                                <p>Wasabi generates Bech32 addresses only, also known as native SegWit addresses or bc1 addresses. Since these addresses start with the characters "bc1…", some wallets/exchanges do not yet support this type of address and may give an error message (e.g. “unknown bitcoin address”). The hack is to manage your funds with a wallet that does support Bech32. To check Bech32 adoption and exchange/wallet support you can follow <a href="#" target="_blank">Bitcoin.it Wiki</a> and <a href="#" target="_blank">When Segwit? website</a>.</p>
+                            </div>
+                        </div>
+                        <div class="card">
+                            <input type="checkbox" id="faq5">
+                            <label for="faq5">
+                                WHAT ARE THE FEES FOR coinjoinS?
+                            </label>
+                            <div class="card-content">
+                                <p>You currently pay a fee of 0.003% * anonymity set. If the anonymity set of a coin is 50 then you pay 0.003% * 50 (=0.15%). If you set the target anonymity set to 53 then Wasabi will continue mixing until this is reached, so you may end up with an anonymity set of say 60, and you will pay 0.003% * 60 (=0.18%). </p>
+
+                                <p>There are also cases where you may not pay the full fee or where you pay more. For example, if you’re the smallest registrant to a round, you will never pay a fee. Also when you are remixing and you cannot pay the full fee with your input, then you only pay as much as you have. But, if the change amount leftover would be too small, then that is also added to the fee. Currently the minimum change amount to be paid out is 0.7% of the base denomination (~0.1BTC.)</p>
+
+                                <p>It is also possible that you get more back from mixing than you put in. This happens when network fees go down between the start of the round and its end. In this case, the difference is split between the active outputs of the mix.</p>
+                            </div>
+                        </div>
+                        <div class="card">
+                            <input type="checkbox" id="faq6">
+                            <label for="faq6">
+                                WHAT IS AN “ANONYMITY SET?
+                            </label>
+                            <div class="card-content">
+                                <p>The anonymity set is essentially the number of participants in a single coinjoin round </p>
+
+                                <p>If 3 people take part in a coinjoin (with equal size inputs) and there are 3 outputs then each of those output coins has an anonymity set of 3.</p>
```

Comment:
```suggestion
                                <p>The anonymity score is essentially the number of participants in a single coinjoin round </p>

                                <p>If 3 people take part in a coinjoin with 3 outputs, then each of those output coins has an anonymity score of 3.</p>
```

Not 100% accurate, highly simplified, but probably good enough.